examples/flutter_gallery/lib/demo/cupertino/cupertino_buttons_demo.dart

```diff
@@ -25,18 +25,18 @@ class _CupertinoButtonDemoState extends State<CupertinoButtonsDemo> {
         actions: <Widget>[MaterialDemoDocumentationButton(CupertinoButtonsDemo.routeName)],
       ),
       body: Column(
-        children: <Widget> [
+        children: <Widget>[
           const Padding(
             padding: EdgeInsets.all(16.0),
             child: Text(
               'iOS themed buttons are flat. They can have borders or backgrounds but '
-              'only when necessary.'
+                  'only when necessary.',
```

Comment:
This is a bad reformat. I think the problem with multiline strings has been brought up already...